### PR TITLE
python3Packages.pnglatex: init at 1.1

### PR DESCRIPTION
**Pull request by GaetanLepage**
###### Description of changes

Add python3 package [pnglatex](https://github.com/MaT1g3R/pnglatex): a small program that converts LaTeX snippets to png.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/pnglatex/default.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    netpbm
+  ];
+
+  # There are no tests
+  doCheck = false;
+
+  meta = with lib; {
+    homepage = "https://github.com/MaT1g3R/pnglatex";
+    description = "a small program that converts LaTeX snippets to png";
+    license = licenses.gpl3;
```
Comment (wegank) [inline pkgs/development/python-modules/pnglatex/default.nix:24]:
```suggestion
    license = licenses.gpl3Plus;
```

File: pkgs/development/python-modules/pnglatex/default.nix [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{ pkgs, lib, buildPythonPackage, fetchPypi }:
```
[the remark this answers is not in the document]
Comment (GaetanLepage) [reply pkgs/development/python-modules/pnglatex/default.nix]: > 

Done !

**Comment by wegank** [comment]
@ofborg build python310Packages.pnglatex

**Review by wegank — APPROVED** [review]

**Comment by wegank** [comment]
Do you want to promote the package to top-level via `toPythonApplication`?

**Comment by GaetanLepage** [comment]
> Do you want to promote the package to top-level via `toPythonApplication`?

Done !

**Comment by GaetanLepage** [comment]
Result of `nixpkgs-review` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>pnglatex (python310Packages.pnglatex)</li>
    <li>pnglatex.dist (python310Packages.pnglatex.dist)</li>
    <li>python311Packages.pnglatex</li>
    <li>python311Packages.pnglatex.dist</li>
  </ul>
</details>

**Comment by wegank** [comment]
@ofborg build pnglatex

**Comment by wegank** [comment]
Result of `nixpkgs-review pr 222735` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>4 packages built:</summary>
  <ul>
    <li>pnglatex (python310Packages.pnglatex)</li>
    <li>pnglatex.dist (python310Packages.pnglatex.dist)</li>
    <li>python311Packages.pnglatex</li>
    <li>python311Packages.pnglatex.dist</li>
  </ul>
</details>

**Review by wegank — APPROVED** [review]

**Pull request merged**